### PR TITLE
adds modified facet_list snippet from base ckan

### PR DESCRIPTION
**Pull request by btylerburton**
includes line to parse bureau codes

### CODE REVIEW

File: ckanext/datagovtheme/templates/templates_new/snippets/facet_list.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                {% block facet_list_items %}
+                {% with items = items or h.get_facet_items_dict(name, search_facets) %}
+                <!-- {{items}} -->
+                    {% if items %}
+                    <nav aria-label="{{ title }}">
+                        <ul class="{{ nav_class or 'list-unstyled nav nav-simple nav-facet' }}">
+                        {% for item in items %}
+                            {% set href = h.remove_url_param(name, item.name, extras=extras, alternative_url=alternative_url) if item.active else h.add_url_param(new_params={name: item.name}, extras=extras, alternative_url=alternative_url) %}
+                            {% set label = label_function(item) if label_function else item.display_name %}
+                                {% if title == 'Bureaus' %}
```
Comment (btylerburton) [inline ckanext/datagovtheme/templates/templates_new/snippets/facet_list.html:69]: Is there something better than a string that I can use to compare here? Or is that fine?

**Review by jbrown-xentity — APPROVED** [review]
All good!
Just fyi, the ckan2.8 tests probably failed because we only added the code to the new template (aka ckan2.9)

**Comment by btylerburton** [comment]
For completeness' sake, I added the snippet to 2.8 folder.

**Comment by btylerburton** [comment]
@jbrown-xentity can you approve again?

Not sure what's going wrong with 2.8. I get other errors related to importing `unicode_safe` in my local and since it's not required I should have just left well-enough alone and merged.

**Review by nickumia-reisys — APPROVED** [review]
Overall, I think this is efficient.  In theory, it would be nice to be able to track our customizations from core ckan, but that's a lot of documentation to keep up with, not to mention that we don't have a method for organizing it as it stands.

⛰️

**Pull request merged**